inheritance-mapper/README.md

```diff
@@ -0,0 +1,72 @@
+## Intent
```

Comment:
Readme is missing the YAML front matter. See https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute